api/src/resources/gram/v1/validation/validateModel.ts

```diff
@@ -3,38 +3,30 @@
  * @exports {function} handler
  */
 import { DataAccessLayer } from "@gram/core/dist/data/dal.js";
-import Model from "@gram/core/dist/data/models/Model.js";
 import { Request, Response } from "express";
 
 export function validateModel(dal: DataAccessLayer) {
   return async (req: Request, res: Response) => {
-    const modelId = req.params.id;
-    let model: Model | null = null;
-    // Get the model from the DAL
-    try {
-      model = await dal.modelService.getById(modelId);
-
-      if (!model) {
-        return res.sendStatus(400);
-      }
-    } catch (error) {
-      return res.sendStatus(400);
+    if (dal.validationEngine.rules.length === 0) {
+      return res
+        .status(404)
```

Comment:
404 here doesn't make sense. It should just return 200 with an empty list. Or are you handling this on the frontend?

Reply:
400 errors usually denote client errors, which this is not.